src/assets/style/helpers/_vars.less

```diff
@@ -17,6 +17,8 @@
 @light-green: rgba(128, 255, 255, 0.3);
 @darkgreen-dark: rgba(15, 25, 20, 1.0);
 @darkgreen-medium: rgba(31, 65, 48, 1.0);
+@blue-medium: rgba(40, 80, 90, 1.0);
+@blue-light: rgb(56, 111, 125);
```

Comment:
just for consistency, can you make this an `rgba`?

Reply:
done